app/js/team.js

```diff
@@ -156,11 +181,42 @@ angular.module('teamform-team-app', ['firebase'])
 				$scope.param.teamMembers = data.child("teamMembers").val();
 				
 			}
+			else{
+				$scope.param.teamMembers = "";
+			}
+
+			if ( data.child("mergeRequests").val() != null){
+				$scope.mergeSelection = data.child("mergeRequests").val();
+			}
 			
 			$scope.$apply(); // force to refresh
 		});
 
 	}
+
+		/*************** TeamMergeRequest *******************/
+	$scope.mergeSelection = [];
+	$scope.teamMergeRequest = function(teamItem){
+		if(typeof teamItem.teamMembers == "undefined"){
+			teamItem.teamMembers = [];
+		}
+		if($scope.param.teamMembers.length + teamItem.teamMembers.length <= teamItem.size && $scope.mergeSelection.length < 1){
```

Comment:
Is that `$scope.mergeSelection.length < 1` controlling team leader to send request to only one team?